botocore/awsrequest.py

```diff
@@ -54,7 +50,7 @@ def _read_status(self):
             return HTTPResponse._read_status(self)
 
 
-class AWSHTTPConnection(HTTPConnection):
+class AWSConnection(object):
     """HTTPConnection that supports Expect 100-continue.
```

Comment:
docstring is out of date.

Reply:
Kind of? The functionality and inheritance described is still true, it just doesn't actually get inherited until it's used as a mixin.